src/main/java/org/terasology/AcceptanceTestEnvironment.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology;
+
+import org.junit.After;
+import org.junit.Test;
+import org.terasology.engine.GameThread;
+import org.terasology.recording.RecordAndReplayStatus;
+
+/**
+ * An environment that extends {@link ReplayTestingEnvironment} and sets the workflow of a replay test used for
+ * acceptance testing. The tests that extends this class instead of {@link ReplayTestingEnvironment} will be more limited
+ * but easier to write.
+ * <p>
+ * To write tests that uses this class, it is necessary to extend it and fill the three abstract methods {@link #testOnReplayStart()} ,
+ * {@link #testDuringReplay()} and {@link #testOnReplayEnd()} . If it is not desired to test something in one or more of
+ * the replay stages, it is possible to leave the methods in black.
```

Comment:
I feel that this paragraph may be better utilized by describing when the three abstract methods will be called and what those method implementations could potentially contain. Right now, this paragraph contains redundant information which is either already enforced by the compiler/IDE ("all three methods need to be overriden"), or would be much better suited in the method specific documentation ("if it is desired, the methods may be blank").